spec/helpers/theme_helper_spec.rb

```diff
@@ -0,0 +1,108 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe ThemeHelper, :type => :helper do
+  describe "#get_hex_color_from_theme_value" do
+    it "returns the proper hex value from the theme value" do
+      expect(helper.get_hex_color_from_theme_value(16777215)).to eq("ffffff")
```

Comment:
Please (also) test this with a value other than `ffffff` -- e.g. just `rgb(0, 0, 255)`